R/get_code.R

```diff
@@ -92,6 +94,47 @@ get_code.TealDataAbstract <- function(x, dataname = character(0), deparse = TRUE
   }
 }
 
+#' @rdname get_code
+#' @export
+#' @examples
+#'
+#' tdata1 <- teal_data()
+#' tdata1 <- within(tdata1, {
+#'   a <- 1
+#'   b <- a^5
+#'   c <- list(x = 2)
+#' })
+#' get_code(tdata1)
+#' get_code(tdata1, "a")
+#' get_code(tdata1, "b")
+#'
+#' tdata2 <- teal_data(x1 = iris, code = "x1 = iris")
+#' get_code(tdata2)
+#' get_code(verify(tdata2))
+#'
+get_code.teal_data <- function(x, names = NULL, deparse = TRUE, ...) {
+  check_ellipsis(...)
+  checkmate::assert_character(names, min.len = 1L, null.ok = TRUE)
+  checkmate::assert_flag(deparse)
+  checkmate::assert_subset(names, x@datanames, empty.ok = TRUE)
```

Comment:
reorder according to formals

Reply:
7ee2270c5b4b0677079b46e244e402c8e2461d83